local-test.sh

```diff
@@ -20,7 +22,7 @@ else
 	EXTRA_MAVEN_PROPERTIES=
 fi
 
-if [[ -n ${DOCKERIZED-} ]]; then
+if [[ "${DOCKERIZED-}" == "true" ]]; then
```

Comment:
No need for quotes around the left-hand side in `bash`:

```suggestion
if [[ ${DOCKERIZED-} == "true" ]]; then
```